esque/clients/consumer.py

```diff
@@ -51,13 +51,19 @@ def _consume_single_message(self, timeout=30) -> Optional[Message]:
 
```

Comment:
Also because raise_for_message raises an exception when the message is none the output can be just Message, not Optional[Message]